lib/pbench/test/functional/server/conftest.py

```diff
@@ -1,12 +1,13 @@
+from http import HTTPStatus
 import os
 
 import pytest
 
-from pbench.client import PbenchServerClient
+from pbench.client import API, PbenchServerClient
 
 
 @pytest.fixture(scope="module")
-def pbench_server_client():
+def server_client():
```

Comment:
This would have been a better comment for the original functional tests, since the `scope` hasn't changed. At the time I know I thought about this, but I no longer remember the logic.